homeassistant/components/flukso/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""The Flukso integration."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+from .const import CONF_DEVICE_FIRMWARE, CONF_DEVICE_HASH, CONF_DEVICE_SERIAL, DOMAIN
+from .discovery import async_get_configs
+
+PLATFORMS: list[str] = ["binary_sensor", "sensor"]
```

Comment:
Note that only in rare cases you can have multiple platforms in your initial PR, you can add other platforms/features in a later PR. -> https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible

Reply:
I understand this guideline to keep the PR code size as small as possible to ease the review process, but in this case, adding only 1 platform instead of 2 would not make much of a difference in code size.